src/l500/l500-controls.h

```diff
@@ -0,0 +1,134 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2018 Intel Corporation. All Rights Reserved.
+
+#pragma once
+#include "hw-monitor.h"
+#include "l500-device.h"
+//#include "option.h"
+
+namespace librealsense
+{
+    enum l500_control
+    {
+        confidence = 0,
```

Comment:
enum values (constants) should follow same convention as in rs_option.h? I.e., all caps?

Also, 'CONFIDENCE' is too broad -- I think we should prefix each enum value with the enum name. So 'L500_OPTION_CONFIDENCE'.